src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -74,168 +55,67 @@ object LocalBackend {
+    val tag: String = s"${this.getClass.getName} [UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.call.name}]"
+    val dockerContainerExecutionDir = s"/root/${backendCall.workflowDescriptor.id.toString}"
+    val containerCallRoot = backendCall.call.docker match {
+      case Some(docker) => Paths.get(dockerContainerExecutionDir).resolve(s"call-${backendCall.call.name}")
+      case None => backendCall.callRootPath
+    }
     def buildDockerRunCommand(image: String): String =
       // -v maps the host workflow executions directory to /root/<workflow id> on the container.
       // -i makes the run interactive, required for the cat and <&0 shenanigans that follow.
-      s"docker run -v ${environment.workflowRoot.toAbsolutePath}:$dockerContainerExecutionDir -i $image"
-
-    // Build the docker run command if docker is defined in the RuntimeAttributes, otherwise just the empty string.
-    val dockerRun = call.docker.map(buildDockerRunCommand).getOrElse("")
-
-    // The aforementioned shenanigans generate standard output and then a bash invocation that takes
-    // commands from standard input.
-    val argv = Seq("/bin/bash", "-c", s"cat $commandFile | $dockerRun /bin/bash <&0")
-    logger.debug("Executing call with argv: " + argv)
-
-    val process = argv.run(ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline))
-
-    // TODO: As currently implemented, this process.destroy() will kill the bash process but *not* its descendants. See ticket DSDEEPB-848.
-    callAbortRegisteringFunction.registrationFunction(AbortFunction(() => process.destroy()))
-    val rc: Int = process.exitValue()
-    Vector(stdoutWriter, stderrWriter) foreach { _.flushAndClose() }
-
-    /**
-     * Return a host absolute file path.
-     */
-    def hostAbsoluteFilePath(pathString: String): String =
-      if (new File(pathString).isAbsolute) pathString else Paths.get(environment.cwd.toAbsolutePath.toString, pathString).toString
-
-    val stderrFileLength = new File(stderrFile.toString).length
-
-    if (call.failOnStderr && stderrFileLength > 0) {
-      Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: stderr has length $stderrFileLength for command: $instantiatedCommandLine"))
-    } else {
-      if (rc == 0) {
-        evaluateCallOutputs(workflowDescriptor, call, hostAbsoluteFilePath, environment.engineFunctions, scopedLookupFunction, interpolateStrings=true)
-      } else if (rc == 143) {
-        // Special case to check for SIGTERM exit code - implying abort:
-        Failure(new TaskAbortedException())
-      } else {
-        Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: return code $rc for command: $instantiatedCommandLine\n\nFull command was: ${argv.mkString(" ")}"))
-      }
+      s"docker run -v ${backendCall.workflowRootPath.toAbsolutePath}:$dockerContainerExecutionDir -i $image"
+
+    backendCall.instantiateCommand match {
+      case Success(instantiatedCommand) =>
```

Comment:
Similar to the comment I made in JesBackend. If it's a success, I'm going to do ... what? Dunno, have to read all these lines and find out.